### ISSUE TITLE
allow access to field value by name inside "beforeDeleteRow" callback

### ISSUE THREAD

**Issue by duynguyen0511** [unclead/yii2-multiple-input#40]
```js
jQuery('.multiple-input').on('beforeDeleteRow', function(){
     // want to grab the row field value by name as declared in TabularInput column name
})
```

![delete](https://cloud.githubusercontent.com/assets/436950/10143676/cb07ef06-65d6-11e5-8270-717c107b80bc.png)

**Comment by unclead** [comment]
Lets think about it. I can add `item` into the event arguments next way
```
.on('beforeDeleteRow', function(e, item)) {

}
```
`item` is the raw for removal (HTML). 
i can not realize how we can achieve the aim that you want. I need ideas :smile:

**Comment by duynguyen0511** [comment]
i am thinking

```js
.on('beforeDeleteRow', function(e, row)) {

}
```

where `row` is the container of the row being delete

![2015-10-01_1012](https://cloud.githubusercontent.com/assets/436950/10226523/2a91a9f8-6825-11e5-955a-67101cee61e2.png)

If i have access to the container, I can traverse down to find the elements

**Comment by unclead** [comment]
Closed via e9a122cf72c7f7c0527c73d489d21480c9ebbfb6
